lib/preprocessCss.js

```diff
@@ -13,28 +12,33 @@ const getFunctionArgs = require('./getFunctionArgs')
 module.exports = function (app, callback) {
   const params = app.get('params')
   const appName = app.get('appName')
-  const preprocessor = params.cssCompiler.nodeModule
   const cssPath = app.get('cssPath')
   const cssCompiledOutput = app.get('cssCompiledOutput')
   const usingWhitelist = !!params.cssCompilerWhitelist
   const logger = require('./logger')(app)
+  const fsr = require('./tools/fsr')(app)
+  let preprocessor
   let cssFiles
   let preprocessorModule
   let preprocessorArgs
   let versionFile
   let versionCode = '/* do not edit; generated automatically by Roosevelt */ '
   let promises = []
 
-  if (params.cssCompiler === 'none') {
+  if (params.cssCompiler === 'none' || params.cssCompiler === null) {
     callback()
     return
   }
 
+  preprocessor = params.cssCompiler.nodeModule
+
   // require preprocessor
   try {
     preprocessorModule = prequire(preprocessor)
   } catch (err) {
-    logger.error(`${appName} failed to include your CSS preprocessor! Please ensure that it is declared properly in your package.json and that it has been properly insalled to node_modules.\n`.red, err)
+    logger.error(`${appName} failed to include your CSS preprocessor! Please ensure that it is declared properly in your package.json and that it has been properly insalled to node_modules.`.red)
```

Comment:
fix "insalled"

Reply:
Change default for CSS compiler to "none" to suppress the errors/warnings. The generator can opt-in to preprocessors.